CHANGELOG.md

```diff
@@ -1,6 +1,44 @@
 # Changelog
 
-## Version 0.17.0 (In development)
+## Version 0.17.0
+
+### Summary
+
+This release marks the completion of UnitaryHACK 2022!
+We had 3 merged pull requests, and a fourth is looking to be merged in the next milestone.
+Along with the great contributions from hackers, this release focused on expanding our ZNE examples to other frontends (Cirq, Braket, and Pennylane), building out learning-based PEC, and improving our benchmarking infrastructure.
+
+
+### All Changes
+
+- Support the latest Cirq version (0.15.0) (@andreamari, gh-1379)
```

Comment:
Just a reminder to remove this if it isn't going to make the release, for consistency.

Reply:
Nvm, I see this is closed, nice work!